internal/commands/accesslist/update_test.go

```diff
@@ -0,0 +1,222 @@
+package accesslist
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+	"github.com/Netflix/go-expect"
+)
+
+func TestAllowedIPUpdateHandler(t *testing.T) {
+	projectID := "projectID"
+	appID := "appID"
+	app := realm.App{
+		ID:          appID,
+		GroupID:     projectID,
+		ClientAppID: "test-abcde",
+		Name:        "test",
+	}
+
+	allowedIPs := []realm.AllowedIP{
+		{ID: "address1", Address: "0.0.0.0", Comment: "comment"},
+		{ID: "address2", Address: "192.1.1.1"},
+		{ID: "address3", Address: "192.158.1.38", Comment: "cool comment"},
+	}
+
+	for _, tc := range []struct {
+		description    string
+		testAddress    string
+		testNewAddress string
+		testComment    string
+	}{
+		{
+			description:    "should return a successful message for a successful update for address and comment",
+			testAddress:    "0.0.0.0",
+			testNewAddress: "2.2.2.2",
+			testComment:    "new comment",
+		},
+		{
+			description:    "should return a successful message for an update with only an address",
```

Comment:
can we add one more test case for updating with only a comment?

with that, let's also add a few `var`s for what gets passed into the mocked client's `AllowedIPUpdate` function and have asserts on those as well